Fix Apex tests initialization

martinrrm opened the pull request:
<!-- DO NOT MODIFY OR DELETE THIS TEMPLATE. IT IS USED IN AUTOMATION. -->

# Bug

<!-- If this is an engineering change or test change only, you do not need an issue. -->
<!-- Find or create an issue in NuGet/Home and paste the full url. -->
<!-- At the maintainers discretion, multiple changes may apply to a single issue, but only when the PRs are all created within a short period of time. -->
Fixes: https://github.com/NuGet/Client.Engineering/issues/3301

## Description
There is an error when initializing this class, removing this line makes Apex tests work again:

https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=12355228&view=results


## PR Checklist

- [x] Meaningful title, helpful description and a linked NuGet/Home issue
- ~[ ] Added tests~ fixes the tests
- [x] Link to an issue or pull request to update docs if this PR changes settings, environment variables, new feature, etc.

martinrrm committed:
Try making apex tests work again

nkolev92 commented on Directory.Packages.props:
```diff
@@ -74,6 +74,10 @@
     <PackageVersion Include="Microsoft.VisualStudio.TemplateWizardInterface" Version="17.10.40170" />
+
+    <PackageVersion Include="EnvDTE" Version="18.0.2345-preview.1" />
+    <PackageVersion Include="Microsoft.VisualStudio.OLE.Interop" Version="18.0.2345-preview.1" />
```
You are not using this anywhere. 

IMO, you should merge the other change and not add risk to this PR.

The E2E tests can be fixed in a follow-up PR.

martinrrm commented:
@nkolev92 @aortiz-msft @jeffkl @zivkan I reverted the changes done to e2e tests and kept this as just for APEX, I rebased also so it should pass now
